plugin/src/main/groovy/com/novoda/staticanalysis/internal/ktlint/KtlintConfigurator.groovy

```diff
@@ -134,7 +136,21 @@ Last tested compatible version: $LAST_COMPATIBLE_KTLINT_VERSION
             File xmlReportFile = null
             File txtReportFile = null
             try {
-                ktlintTask.reportOutputFiles.each { key, fileProp ->
+                Map<String, RegularFileProperty> outputFiles = null
+
+                /**
+                 * This change was made to support version 9.0.0 of the JLLeitschuh/ktlint-gradle plugin which renamed
+                 * this property: https://github.com/JLLeitschuh/ktlint-gradle/blob/master/CHANGELOG.md#900---2019-09-30
+                 */
+                if (ktlintTask.hasProperty("allReportsOutputFiles")) {
```

Comment:
Because of its nature, this code class is not the cleanest. We can help it more clean by moving this into a private function below. The comment can go above the function then. What do you think?